pkgs/development/python-modules/paddlepaddle/binary-hashes.nix

```diff
@@ -0,0 +1,34 @@
+{ pythonVersion, system }:
+let
+  all = {
+    x86_64-linux = {
+      platform = "manylinux1_x86_64";
+      hashes = {
+        cp39 = "sha256-Yu/FWoMhYp+behAth/jH0FKlf2LJr8TyvL9MBwmuews=";
+        cp310 = "sha256-O7d/5LY2dEMf5gW5WrN3xzIIEi2vT0RWoMeVOk5lATk=";
+      };
+    };
+    x86_64-darwin = {
+      platform = "macosx_10_9_x86_64";
+      hashes = {
+        cp39 = "sha256-5g9b2gC6uosMpoJiobpj8yToIS6ifAFRvLEqnc/o/QQ=";
+        cp310 = "sha256-2c1hjwNCOOOx9tVfBk+Pyk/pF0m/2tAmRsBH91834eM=";
+      };
+    };
+    aarch64-darwin = {
+      platform = "macosx_11_0_arm64";
+      hashes = {
+        cp39 = "sha256-JhYNTOx1UkuNf/63lHXBDry6FQjPnbIB8jU5jKcyX2k=";
+        cp310 = "sha256-4ltYEYm2OzPBc6D2bQt2dEh6Sz+5m1mMKGGYgQGLSAY=";
+      };
+    };
+
+    # Satisfy ofborg:
+    aarch64-linux = {platform = ""; hashes = {cp39 = ""; cp310 = "";};};
+  };
+  selected = all."${system}";
+in
+{
+  hash = selected.hashes."${pythonVersion}";
+  platform = selected.platform;
+}
```

Comment:
```suggestion
  };
in
all
```

Reply:
my suggestion here is what if you left the binary hashes as just an attrset that contains everything.
(removing the aarch64-linux in the process).

Then "selected" the right hash and platform based on the attributes inside the paddlepaddle file.

I'm not sure why, but ofborg is still evaluating aarch64-linux, we have missed something.
I'm trying to think about the order of evaluation to understand where we have missed something.

Reply:
Maybe ofborg is evaluating our `src` property for all systems, even though it has no intention of actually instantiating the derivation?

Reply:
that shouldn't happen though, nix evaluation is lazy and so nothing that isn't used is evaluated. ofborg is evaluating because it wants to use it.
We are missing something.
I'm going to run a few tests this weekend.